src/main/java/InterfaceFigure.java

```diff
@@ -0,0 +1,3 @@
+public interface InterfaceFigure {
+    double getSquire();
```

Comment:
Better `calculateSquire`, because usually getter uses for fields without mathematics